README.md

```diff
@@ -17,6 +17,10 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 
 To run a complete build on command line only, execute `build.cmd` or `build.sh` without arguments. See [developer documentation](https://github.com/aspnet/Home/wiki) for more details.
 
+## File logging for functional test
+
+Turn on file logging for Kestrel functional tests by specifying the environment variable ASPNETCORE_TEST_LOG_DIR to the log output directory.
```

Comment:
Why not make this enabled by default? Are we going to make sure the environment variable get set on travis/appveryor/teamcity?

Reply:
Yes we have to set it on all the build configurations. The file logger looks for the environment variable to write the logs to file. If we want this to be automatic, then we need to update the logger to choose a default location. File an issue to Logging if we want that?